docs/src/tut1.rst

```diff
@@ -50,17 +50,21 @@ as well as words that only appear once in the corpus:
     >>>
     >>> # remove common words and tokenize
     >>> stoplist = set('for a of the and to in'.split())
-    >>> texts = [[word for word in document.lower().split() if word not in stoplist]
-    >>>          for document in documents]
+    >>> texts = [
+    >>>     [word for word in document.lower().split() if word not in stoplist]
```

Comment:
Do we want to demonstrate the best practice for string tokenization here?

Reply:
The best practice is to use a dedicated library (e.g. NLTK)… do we want to complicate the tutorial?

Maybe we should.

But if we don't, this should be something really stupid (like it is now), so people don't accidentally copy-paste thinking it's a good idea.

Reply:
I was thinking more along the lines of https://radimrehurek.com/gensim/utils.html#gensim.utils.tokenize

WDYT?

Reply:
OK, that's an option (but `utils.simple_preprocess` better),